Sources/AppStoreConnectCLI/Services/Operations/GetBetaTestersInfoOperation.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct GetBetaTestersInfoOperation: APIOperation {
+
+    private enum GetTestersError: LocalizedError {
```

Comment:
Still wondering if we should just start calling these operations `Error` and have the execute signature say `Swift.Error`. May as well leave it until we decide either way

Reply:
Already did this on other PRs so I will just change this to Error and Swift.Error